OpenEdXMobile/src/main/java/org/edx/mobile/util/JavaUtil.java

```diff
@@ -82,7 +82,7 @@ public static Double formatDoubleValue(@NonNull Double value, int places) {
      */
     @NonNull
     public static String truncateString(@NonNull String string, int maxChars) {
-        if (string.length() > maxChars) {
+        if (string != null && string.length() > maxChars) {
```

Comment:
I am not sure if the not null check was needed, cuz the function always accepts and returns a NonNull.